### ISSUE TITLE
Updating podcasts over VPN is not working

### ISSUE THREAD

**Issue by petrhejna** [y20k/escapepod#19]
Hi, I noticed that when I am connected to VPN (namely NordVPN) the loading bar at top goes forever without actually downloading the new stuff.

Once I disconnect VPN all updates run smoothly.

******

I am not sure what is the expected behavior, as the VPN can be blocked by podcast source etc...

But maybe I would be useful to add some better error-handling to communicate to the user which podcast cannot be downloaded and download the rest.

Thank you

**Comment by petrhejna** [comment]
I believe you can reproduce it on this podcast:

https://bitcoinundco.com/en/feed/mp3/


(But it was wierd it started to work for me, and when I removed the podcast and re-added it, it happend again.)

**Comment by y20k** [comment]
Hi @petrhejna,

I think, I found the issue. When you add a podcast, Escapepod automatically puts the first episode to the download queue - the red progress bar indicates this. But downloads only start, when a user is connected to WiFi. Escapepod cannot in all cases determine, if a VPN runs run on top of a WiFi or a Cell connection (see #17).

Just a side note: The issue can be reproduced without a VPN, too. :-) Just add a new podcast, when you are only using a Cell connection.

Anyway. I changed the behavior of the app, so that it will only put an episode automatically in the download queue, when it can detect a WiFi connection. That should fix the problem with the red progress indicator.

Users on cell connections - and on certain VPN connections (#17) - need to download episodes manually, though.
